src/index.js

```diff
@@ -51,11 +51,25 @@ async function run() {
 
   if (comment) {
     const message = generateTable(metric);
-    client.issues.createComment({
+
+    const { data: existingComments } = await client.issues.listComments({
       ...context.repo,
       issue_number: prNumber,
-      body: message,
     });
+
+    if (existingComments.length > 0) {
+      client.issues.updateComment({
+        ...context.repo,
+        comment_id: existingComments[0].id,
```

Comment:
First of all, thanks for the PR. I am afraid this may change an undesirable comment, the coverage report is not necessarily the first comment in PR, is it?

Reply:
My understanding is that the `context.repo` object has the "owner" set to the runner of this github action and that would have filtered comments down to this action- I will verify

Reply:
Hey @luciano7,

JFYI, I've the `listComments` returns all comments in the issue, I even was able to remove all of them since GITHUB_TOKEN has all needed permissions.

Anyways, thank you again for idea, I make some changes in #10, now we will not be spammed by bot 🤣

Reply:
crud - I haven't had any time to try it- thank you so much for verifying!